components/objectKey/ObjectKeyHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace app\components\objectKey;
+
+use app\models\Comment;
+use app\models\Extension;
+use app\models\File;
+use app\models\News;
+use app\models\Wiki;
+use yii\base\InvalidValueException;
+
+class ObjectKeyHelper
+{
+    const TYPE_NEWS = 'news';
+    const TYPE_WIKI = 'wiki';
+    const TYPE_EXTENSION = 'extension';
+    const TYPE_COMMENT = 'comment';
+    const TYPE_FILE = 'file';
+
+    const TYPE_GUIDE = 'guide';
+    const TYPE_API = 'api';
+
+    public static $typeClasses = [
```

Comment:
Could be renamed to `$types`.

Reply:
A variable returns classes. Maybe `$classes`?